mypy/stubutil.py

```diff
@@ -669,8 +669,6 @@ def set_defined_names(self, defined_names: set[str]) -> None:
                 self.add_name(f"{pkg}.{t}", require=False)
 
     def check_undefined_names(self) -> None:
-        print(self._all_)
-        print(self._toplevel_names)
```

Comment:
These spurious prints are not related but they seem to be left here by mistake. They are polluting the stdout.
The blame points to #15770.